qa/rpc-tests/fundrawtransaction.py

```diff
@@ -678,14 +678,18 @@ def run_test(self):
         assert(signedtx["complete"])
         self.nodes[0].sendrawtransaction(signedtx["hex"])
 
+        self.sync_all()
```

Comment:
Nit: You don't need the sync when you mine on the node where you broadcast the transaction.